### ISSUE TITLE
Android doesn't boot after applying fex patches

### ISSUE THREAD

**Issue by by2coffee** [BPI-SINOVOIP/BPI-M2-bsp#4]
I followed the steps in README to recompile and dd all the fex files as listed. But the Android OS doesn't boot any longer.

Using the serial console (UART), I can see that the boot stops with this error:

`boota: bad boot image magic, maybe not a boot.img?
try to read all
[      5.051]sunxi flash read :offset 4000000, 16777216 bytes OK
no signature
boota: bad boot image magic, maybe not a boot.img?`

And it enters the `sunxi#` prompt. 

How to proceed?

**Comment by McGr3g0r** [comment]
Well,
The boot.img is located under sdcard offset 86016K (it's boot.fex), you may need to check if that area isn't corrupted, or just reflash it.

The README also does not mention the sys_config.bin flashing, which you may need sometimes to change and recompile - the offset is 19848K.

Best Regards,
McGr3g0r

**Comment by by2coffee** [comment]
I did overwrite 86016K with new boot.fex, as per the README. Did you mean something else? 

Thanks, I'll try overwriting sys_config.bin, but where do I find it (or how to I compile it)?

**Comment by by2coffee** [comment]
Nope, I doubled checked if I had overwritten boot.fex, and I am still unable to boot into Ubuntu (or Android)
